docker/install.sh

```diff
@@ -86,14 +70,44 @@ if [ $build_ros1_pkgs -eq 1 ]; then
 
     exit #Success building ros1 pkgs
 
-elif [ $build_ros2_pkgs -eq 1 ]; then
+elif [[ $build_ros2_pkgs -eq 1 ]]; then
 
-    cd ~/workspace_ros2
-    source /opt/autoware.ai/ros/install_ros2/setup.bash
+    if [ -z $token ];
+        then
+            echo "No argument provided for token for ROS2 build, this script needs to be run with <TOKEN>"
+            exit 1
+    fi
+    cd ~
+    source /opt/ros/humble/install/setup.bash
+    git clone https://$token@github.com/usdot-fhwa-stol/CARMASensitive.git --branch arc-199-humble-lexus-ssc-deb-files
```

Comment:
Can we update the CARMA Sensitive branch here to point to develop? Alternatively this can also be an argument

Reply:
switched